### ISSUE TITLE
Cant start kolide agent on OS-X

### ISSUE THREAD

**Issue by moivica** [kolide/launcher#813]
Hello ! We are testing Kolide  and want to use it in our company. 
But we are having problem with installation. 

Installing kolide agent (personal device mode) on OS-x (monterey 12.2.1) does not work.  Installation went smooth, but after installation nothing happened. When checking kolide bot messages in slack , it stands : `Waiting for your device to connect...`

Here are logs from  `kolide-k2/launcher-stderr.log`: 

```
{"caller":"","component":"osquery","msg":"[/usr/local/kolide-k2/bin/osquery-extension.ext --socket /var/kolide-k2/k2device.kolide.com/osquery.sock --timeout 10 --interval 3]","osqlevel":"stderr","severity":"info","ts":"2022-04-07T09:09:05.224283Z"}
{"caller":"interface.cpp:137","component":"osquery","msg":"I0407 11:09:05.575292 137945088 interface.cpp:137] Registering extension (kolide, 47904, version=, sdk=)","osqlevel":"stderr","severity":"info","ts":"2022-04-07T09:09:05.575527Z"}
{"caller":"launcher.go:126","err":"enrolling host: query enrollment details, (even with retries): done trying: query enrollment details: could not query the extension manager client: read unix -\u003e/var/kolide-k2/k2device.kolide.com/osquery.sock: i/o timeout","msg":"interrupted","severity":"info","ts":"2022-04-07T09:09:32.890435Z"}
{"caller":"extension.go:162","err":"enrolling host: query enrollment details, (even with retries): done trying: query enrollment details: could not query the extension manager client: read unix -\u003e/var/kolide-k2/k2device.kolide.com/osquery.sock: i/o timeout","msg":"extension interrupted","severity":"info","ts":"2022-04-07T09:09:32.892387Z"}
{"caller":"launcher.go:122","msg":"beginnning shutdown via signal","severity":"info","ts":"2022-04-07T09:09:32.892471Z"}
{"caller":"control.go:42","err":"enrolling host: query enrollment details, (even with retries): done trying: query enrollment details: could not query the extension manager client: read unix -\u003e/var/kolide-k2/k2device.kolide.com/osquery.sock: i/o timeout","msg":"control interrupted","severity":"info","ts":"2022-04-07T09:09:41.924573Z"}
{"caller":"updater.go:112","err":"enrolling host: query enrollment details, (even with retries): done trying: query enrollment details: could not query the extension manager client: read unix -\u003e/var/kolide-k2/k2device.kolide.com/osquery.sock: i/o timeout","msg":"updater interrupted","severity":"info","ts":"2022-04-07T09:09:41.925538Z"}
{"caller":"updater.go:112","err":"enrolling host: query enrollment details, (even with retries): done trying: query enrollment details: could not query the extension manager client: read unix -\u003e/var/kolide-k2/k2device.kolide.com/osquery.sock: i/o timeout","msg":"updater interrupted","severity":"info","ts":"2022-04-07T09:09:41.926372Z"}
{"caller":"swap_signal.go:26","debug":true,"msg":"swapping level","severity":"info","ts":"2022-04-07T09:19:22.53608Z"}
```

I've tried with uninstall (used steps from your offical web page) and install again, but it's same. 

 Can u please help?

**Comment by directionless** [comment]
Hi There! 

Generally speaking, the best route for product support is to reach out to support via the intercom chat widget.

But since you're here... This looks like an issue I'm tracking that effects a small number of Monterey users.  Do you know what model of mac you have -- is an m1 or intel machine?

**Comment by moivica** [comment]
Hello, im using intel machine:  `2,9 GHz Quad-Core Intel Core i7`

**Comment by sheagcraig** [comment]
We are seeing similar errors across numerous machines in our environment; all of ours are M1s.

Do you have any recommendations for things we can try?

**Comment by sheagcraig** [comment]
Thanks-we grabbed that and tentatively it seems to resolve the issues. I'll update here if we get more specifics and it's not working.

**Comment by directionless** [comment]
I believe this was resolved in the 0.12 line.
